docs/glues/herbs2knex.md

```diff
@@ -124,6 +124,25 @@ class YourRepository extends Repository {
     ids: [`customerId`, `productId`]  // productItem.customerId , productItem.productId
     ```
 
+    It's is possible to omit this field if the entity already has fields defined on it with `field` and the `isId` option or `id` helper functions:
+
+    ```javascript
+    // entity definition
+    const Order = entity('Order', {
+        orderSequence: field(Number, { isId: true }),
+        customerId: id(Number)
+    })
+
+    // repository definition
+    class OrderRepository extends Repository {
+        constructor({
+            entity: Order,
+            ids: ['orderSequence', 'customerId'] // `this` field is optional, since herbs can discover it through the entity metadata
```

Comment:
Suggestion:

- Change the text to: `It's is possible to omit the ``ids`` property if the entity already...`

- Change the line with ids 
from: `ids: ['orderSequence', 'customerId'] // this field is optional, since herbs can discover it through the entity metadata`
to: ` // same as ids: ['orderSequence', 'customerId']`

Reply:
Sorry about the long waiting time. Suggestions applied.